KnowSheet/scripts/KnowSheetReadlink.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# `readlink -e` replacement compatible with Mac OS X `readlink` implementation.
+# Credits to http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in and https://github.com/tarruda/zsh-autosuggestions/issues/38#issuecomment-69258372
+
+set -u -e
+
+SOURCE="$1"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
```

Comment:
Google-level nits: 1) Two spaces before the `#`, please, and 2) Line too long to fit GitHub's review pane, please break it.